src/Intersection.jl

```diff
@@ -308,6 +308,11 @@ or a hyperplane in a given direction.
     * `"line_search"` -- solve the associated univariate optimization problem
                          using a line search method (either Brent or the
                          Golden Section method) 
+    * `"projection"`  -- evaluate the support function by reducing the problem to
+                         the 2D intersection of a rank 2 linear transformation of
+                         the given compat set in the plane generated by the given
+                         direction `d` and the hyperplane's normal vector `n`;
+                         only valid for intersection with a hyperplane   
```

Comment:
Maybe move the last sentence to the top; otherwise the `hyperplane` in the text is confusing.